views/demo/classes.njk

```diff
@@ -3,5 +3,5 @@
 {% block content %}
     {% include "partials/nav-account.njk" %}
 
-    <div class="mariana-reservations-all-binding"></div>
+    <div class="mariana-reservations-all-binding" data-mariana-schedule-redirect-url="/schedule/#/week/"></div>
```

Comment:
here's how to make your link work throughout the app: https://github.com/Mariana-Tek/express-mariana-components-demo/blob/master/views/demo/schedule.njk#L5

Reply:
hey so I added those but the problem seems to be that there is no `locationId` when viewing reservations from the account page.. so when you click "Add guest" while editing a reservation on there, it tries to redirect to /schedule//#/week/324, notice the // because locationId is undefined. it's definitely the source of this bug https://marianatek.atlassian.net/browse/WEB-357